autogen/main/variables.tf.tmpl

```diff
@@ -332,8 +332,8 @@ variable "cluster_telemetry_type" {
   description = "Available options include ENABLED, DISABLED, and SYSTEM_ONLY"
   default     = null
 }
-
 {% endif %}
+
```

Comment:
It seems the same code to me. If this new line is not for particular purpose, could you please revert. This could help to keep changes clean.